lib/restforce_mock/sandbox.rb

```diff
@@ -18,7 +18,6 @@ def self.get_object(name, id)
 
     def self.update_object(name, id, attrs)
       current = storage[name][id]
-      validate_all_present_fields!(current, attrs)
```

Comment:
current validation is not correct, so it will produce wrong results. so i decided to remove it until there is a better way